### DISCUSSION TITLE
Okd 4.8 deployment got failed

### DISCUSSION THREAD

**Discussion by riteshmishra00** [okd-project/okd#915] [Q&A]
In my case bootstrap is working fine but whenever we're provisioning the master node, we're getting unknown error.
api-int.okd.inosys.com is connecting to IPv6 by default.
FCOS version: fedora-coreos-34.20210904.3.0-live.x86_64.iso
OKD Version: 4.8
Openshift Client Version: Client Version: 4.8.0-0.okd-2021-10-01-221835
Openshift Install Version: openshift-install 4.8.0-0.okd-2021-10-01-221835
built from commit 225720cc2adce8c2ee18aea45d474f442cbe0f78
release image quay.io/openshift/okd@sha256:8fac6b281a9a319be5134955403f58b8c1644816ec9f7d28521276ec2ee1e2d3

We have also tried to install OKD 4.6 and 4.7 using FCOS version 35,34 and 33 but everywhere it is throwing the same issue.

Please help me on this if you can.
![136384614-d66ac207-6923-46d2-8f0d-056df320c546](https://user-images.githubusercontent.com/40774769/136441894-aeac7b35-e5b9-4fe8-9c55-107ab30a3d58.png)

**Comment by riteshmishra00** [comment]
apiVersion: v1
baseDomain: Infosys.com
metadata:
  name: okd

compute:
- hyperthreading: Enabled
  name: worker
  replicas: 0

controlPlane:
  hyperthreading: Enabled
  name: master
  replicas: 3

networking:
  clusterNetwork:
  - cidr: 10.128.0.0/14 
    hostPrefix: 23 
  networkType: OpenShiftSDN
  serviceNetwork: 
  - 172.30.0.0/16

platform:
  none: {}

fips: false

pullSecret: '{"auths":{"fake":{"auth": "bar"}}}' 
sshKey: 'ssh-ed25519 AAAA...'

**Comment by vrutkovs** [comment]
Masters can't resolve LB address due to invalid DHCP settings

**Reply by ravimaha9590** [reply]
Hi @vrutkovs 
Do you want us to stop the DHCP service?. 
To set up OKD we are using only static IPs.
